lib/transforms/smart-commit.js

```diff
@@ -1,100 +1,95 @@
-// RDVCS Connector regexp provided by Atlassian
```

Comment:
Same file name, entirely different contents. If you're not already, switch to a "split diff" for a better review experience.